assets/js/src/core/components/tree/components/context-menu/asset-tree-context-menu.tsx

```diff
@@ -78,14 +224,27 @@ export const AssetTreeContextMenu = (props: AssetTreeContextMenuProps): React.JS
 
   return (
     <>
+      <InputModal
+        initialValues={ defaultValue }
+        label={ 'Please enter the new name' }
```

Comment:
Please add translation keys for those messages

Reply:
same here im not sure if it will stay like that in the followup PRs